src/marqo/vespa/models/feed_response.py

```diff
@@ -4,17 +4,14 @@
 
 
 class FeedDocumentResponse(BaseModel):
-    path_id: str = Field(alias='pathId')
-    id: str
-
-
-class FeedBatchDocumentResponse(BaseModel):
```

Comment:
Why are we merging these two?

Reply:
I think they are duplicated because even for `FeedBatchDocumentResponse`, we feed one single document in one semaphore. So the response format is the same. The only different between batch_feed and feed is how we handle the return errors: 1) raise the error directly when feeding one document and 2) catch the error and return the error message on the specific document when a batch of documents

Reply:
Reverted this change